horovod/spark/common/store.py

```diff
@@ -431,3 +445,41 @@ def _check_url(self, url, prefix, path):
     @classmethod
     def filesystem_prefix(cls):
         return cls.FS_PREFIX
+
+
+class DBFSLocalStore(LocalStore):
+    """Uses Databricks File System (DBFS) local file APIs as a store of intermediate data and
+    training artifacts.
+
+    Initialized from a `prefix_path` starts with `/dbfs/...`, see
+    https://docs.databricks.com/data/databricks-file-system.html#local-file-apis.
+
+    Note that DBFSLocalStore cannot be created via Store.create("/dbfs/..."), which will create a
+    LocalStore instead.
+    """
+    def __init__(self, prefix_path, *args, **kwargs):
+        if not prefix_path.startswith("/dbfs/"):
+            warnings.warn("The provided prefix_path might be ephemeral: {} Please provide a "
+                          "`prefix_path` starting with `/dbfs/...`".format(prefix_path))
+        super(DBFSLocalStore, self).__init__(prefix_path, *args, **kwargs)
+
+    def get_checkpoint_filename(self):
+        # Use the default Tensorflow SavedModel format in TF 2.x. In TF 1.x, the SavedModel format
+        # is used by providing `save_weights_only=True` to the ModelCheckpoint() callback.
+        return 'checkpoint.tf'
+
+    def read_serialized_keras_model(self, ckpt_path, model):
+        """
+        Returns serialized keras model. On Databricks, only TFKeras is supported, not BareKeras.
+        The parameter `model` is for providing the model structure when the checkpoint file only
+        contains model weights.
+        """
+        import tensorflow
```

Comment:
Is the reason for having this method unique to this store that it will not work with BareKeras?  If so, we can move this out once we drop BareKeras support.